examples/spectrahedra/data/README.md

```diff
@@ -0,0 +1,35 @@
+## SDPA format
```

Comment:
I think it would be better to put it `examples/spectrahedra` and also explain who compile and run an example